style.css

```diff
@@ -0,0 +1,216 @@
+/* Global Style */
+* {
+  margin: 0;
+  width: 99%;
+  background-image: url('./Image/Overlay.svg');
+  font-style: normal;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+/* Log */
+.log {
+  width: 85%;
+  margin-left: 22px;
+  text-decoration: none;
+  font-size: 18px;
+  font-weight: bolder;
+  text-align: left;
+  color: #9390ff;
+}
+
+/* Icon-navigation */
+.icon-navigation {
+  font-size: 25px;
+}
+
+/* Header and item navigation */
+.top-navigation {
+  width: 95%;
+  height: 15vh;
+  margin-top: 1%;
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+}
+
+/* Item navigation */
+.item-navigation {
+  display: flex;
+  flex-direction: row;
+  flex: 2;
+  gap: 2px;
+}
+
+/* Image of circle */
+.image-circle {
+  width: 10%;
+  height: 15vh;
+  margin-left: 0%;
+}
+
+/* Section container */
+.container-section {
+  display: flex;
+  flex: row;
+}
+
+/* header container */
+.container-header {
+  width: 98.9%;
+  align-items: center;
+}
+
+/* Section Header */
+.header-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 72px;
+  color: #9391ff;
+  margin-top: 2%;
+  text-align: center;
+}
+
+/* Support text of header */
+.support-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: lighter;
+  font-size: 32px;
+  line-height: 64px;
+  text-align: center;
+}
```

Comment:
Both of your titles don't quite look like the ones from the template design.

Here are your titles:

<img width="192" alt="Screen Shot 2022-05-27 at 16 08 05" src="https://user-images.githubusercontent.com/82426682/170789638-2f8ac2d8-9a40-4ba0-a9a2-70a1d868cdc7.png">

Here are the template design titles:

<img width="226" alt="Screen Shot 2022-05-27 at 16 08 33" src="https://user-images.githubusercontent.com/82426682/170789684-31500d70-f0bd-43d2-97a6-acb8a3101ba1.png">

Do you see how different they are in size and how they are styled in normal font as opposed to _italic_? Kindly fix this as following the template as close as possible is a project requirement.